// gatsby-theme-portfolio-minimal/src/components/images5.ts
import { Image } from "react-grid-gallery";

export interface CustomImage extends Image {
  original: string;
}

export const images: CustomImage[] = [
  {
    src: "https://live.staticflickr.com/65535/52566497652_6beb4a20fb_c.jpg",
    original: "https://live.staticflickr.com/65535/52566497652_6beb4a20fb_c.jpg",
    caption: "Step Back for Safety Series Flier",
  },
  {
    src: "https://live.staticflickr.com/65535/52566497647_750c1b4bd9_b.jpg",
    original: "https://live.staticflickr.com/65535/52566497647_750c1b4bd9_b.jpgg",
    caption: "Art for Arts Sake Atish Mukhopadhyay Flt\ier",
  },
  {
    src: "https://live.staticflickr.com/65535/52566961691_4bb9024c95_c.jpg",
    original: "https://live.staticflickr.com/65535/52566961691_4bb9024c95_c.jpg",
    caption: "Louisiana Tip Jar Flier",
  },
  {
    src: "https://live.staticflickr.com/65535/52567484888_a7a2b195f1_c.jpg",
    original: "https://live.staticflickr.com/65535/52567484888_a7a2b195f1_c.jpg",
    caption: "Tanka Writing Workshop",
  },
  {
    src: "https://live.staticflickr.com/65535/52566497782_e6ee95a871_c.jpg",
    original: "https://live.staticflickr.com/65535/52566497782_e6ee95a871_c.jpg",
    caption: "Mod Dance Party Anniversary Flier",
  },
  {
    src: "https://live.staticflickr.com/65535/52567484968_b69a1519b0_c.jpg",
    original: "https://live.staticflickr.com/65535/52567484968_b69a1519b0_c.jpg",
    caption: "Moxie Signs Catalog",
  },
  {
    src: "https://live.staticflickr.com/65535/52567405980_3bce4261b8_c.jpg",
    original: "https://live.staticflickr.com/65535/52567405980_3bce4261b8_c.jpg",
    caption: "Moxie Nola Productions Flyer",
  },
  {
    src: "https://live.staticflickr.com/65535/52567485078_d797ff296a_c.jpg",
    original: "https://live.staticflickr.com/65535/52567485078_d797ff296a_c.jpgg",
    caption: "Moxie Nola Productions Trifold",
  },
  {
    src: "https://live.staticflickr.com/65535/52566961731_a8e81a4017_c.jpg",
    original: "https://live.staticflickr.com/65535/52566961731_a8e81a4017_c.jpg",
    caption: "Faubourg Quartet Program",
  },
  {
    src: "https://live.staticflickr.com/65535/52567236779_d6f5d56de7_c.jpg",
    original: "https://live.staticflickr.com/65535/52567236779_d6f5d56de7_c.jpg",
    caption: "Vessel & Shiop Security Officer Flier",
  },
  {
    src: "https://c7.staticflickr.com/9/8785/28687743710_3580fcb5f0_b.jpg",
    original: "https://c7.staticflickr.com/9/8785/28687743710_3580fcb5f0_b.jpg",
    alt: "Red Zone - Paris",
    width: 320,
    height: 113,
    tags: [{ value: "People", title: "People" }],
    caption: "Red Zone - Paris (Tom Eversley - isorepublic.com)",
  },
  {
    src: "https://c6.staticflickr.com/9/8520/28357073053_cafcb3da6f_b.jpg",
    original: "https://c6.staticflickr.com/9/8520/28357073053_cafcb3da6f_b.jpg",
    alt: "Wood Glass",
    width: 313,
    height: 320,
    caption: "Wood Glass (Tom Eversley - isorepublic.com)",
  },
  {
    src: "https://c8.staticflickr.com/9/8104/28973555735_ae7c208970_b.jpg",
    original: "https://c8.staticflickr.com/9/8104/28973555735_ae7c208970_b.jpg",
    width: 320,
    height: 213,
    caption: "Flower Interior Macro (Tom Eversley - isorepublic.com)",
  },
  {
    src: "https://c4.staticflickr.com/9/8562/28897228731_ff4447ef5f_b.jpg",
    original: "https://c4.staticflickr.com/9/8562/28897228731_ff4447ef5f_b.jpg",
    width: 320,
    height: 194,
    caption: "Old Barn (Tom Eversley - isorepublic.com)",
  },
  {
    src: "https://c2.staticflickr.com/8/7577/28973580825_d8f541ba3f_b.jpg",
    original: "https://c2.staticflickr.com/8/7577/28973580825_d8f541ba3f_b.jpg",
    alt: "Cosmos Flower",
    width: 320,
    height: 213,
    caption: "Cosmos Flower Macro (Tom Eversley - isorepublic.com)",
  },
  {
    src: "https://c7.staticflickr.com/9/8106/28941228886_86d1450016_b.jpg",
    original: "https://c7.staticflickr.com/9/8106/28941228886_86d1450016_b.jpg",
    width: 271,
    height: 320,
    caption: "Orange Macro (Tom Eversley - isorepublic.com)",
  },
  {
    src: "https://c1.staticflickr.com/9/8330/28941240416_71d2a7af8e_b.jpg",
    original: "https://c1.staticflickr.com/9/8330/28941240416_71d2a7af8e_b.jpg",
    width: 320,
    height: 213,
    tags: [
      { value: "Nature", title: "Nature" },
      { value: "People", title: "People" },
    ],
    caption: "Surfer Sunset (Tom Eversley - isorepublic.com)",
  },
  {
    src: "https://c1.staticflickr.com/9/8707/28868704912_cba5c6600e_b.jpg",
    original: "https://c1.staticflickr.com/9/8707/28868704912_cba5c6600e_b.jpg",
    width: 320,
    height: 213,
    tags: [
      { value: "People", title: "People" },
      { value: "Sport", title: "Sport" },
    ],
    caption: "Man on BMX (Tom Eversley - isorepublic.com)",
  },
  {
    src: "https://c4.staticflickr.com/9/8578/28357117603_97a8233cf5_b.jpg",
    original: "https://c4.staticflickr.com/9/8578/28357117603_97a8233cf5_b.jpg",
    width: 320,
    height: 213,
    caption: "Ropeman - Thailand (Tom Eversley - isorepublic.com)",
  },
  {
    src: "https://c4.staticflickr.com/8/7476/28973628875_069e938525_b.jpg",
    original: "https://c4.staticflickr.com/8/7476/28973628875_069e938525_b.jpg",
    width: 320,
    height: 213,
    caption: "Time to Think (Tom Eversley - isorepublic.com)",
  },
  {
    src: "https://c6.staticflickr.com/9/8593/28357129133_f04c73bf1e_b.jpg",
    original: "https://c6.staticflickr.com/9/8593/28357129133_f04c73bf1e_b.jpg",
    width: 320,
    height: 179,
    tags: [
      { value: "Nature", title: "Nature" },
      { value: "Fauna", title: "Fauna" },
    ],
    caption: "Untitled (Jan Vasek - jeshoots.com)",
  },
  {
    src: "https://c6.staticflickr.com/9/8893/28897116141_641b88e342_b.jpg",
    original: "https://c6.staticflickr.com/9/8893/28897116141_641b88e342_b.jpg",
    width: 320,
    height: 215,
    tags: [{ value: "People", title: "People" }],
    caption: "Untitled (moveast.me)",
  },
  {
    src: "https://c1.staticflickr.com/9/8056/28354485944_148d6a5fc1_b.jpg",
    original: "https://c1.staticflickr.com/9/8056/28354485944_148d6a5fc1_b.jpg",
    width: 257,
    height: 320,
    caption: "A photo by 贝莉儿 NG. (unsplash.com)",
  },
  {
    src: "https://c7.staticflickr.com/9/8824/28868764222_19f3b30773_b.jpg",
    original: "https://c7.staticflickr.com/9/8824/28868764222_19f3b30773_b.jpg",
    width: 226,
    height: 320,
    caption: "A photo by Matthew Wiebe. (unsplash.com)",
  },
];
