// gatsby-theme-portfolio-minimal/src/components/images3.ts
import { Image } from "react-grid-gallery";

export interface CustomImage extends Image {
  original: string;
}

export const images: CustomImage[] = [
  {
    src: "https://live.staticflickr.com/65535/52566497777_7f3b05621e_k.jpg",
    original: "https://live.staticflickr.com/65535/52566497777_7f3b05621e_k.jpg",
    caption: "Moxie Nola Productions (Wordpress with Custom child Theme)",
    link: "https://www.moxienola.com/",
    tags: [
      { value: "Link", title: "Link" },
    ],
  },
  {
    src: "https://live.staticflickr.com/65535/52566497892_37c9637e09_b.jpg",
    original: "https://live.staticflickr.com/65535/52566497892_37c9637e09_b.jpg",
    caption: "CBES Ergonomics (Bootstrap)",
    link: "https://cbes-ergonomics.com",
    tags: [
      { value: "Link", title: "Link" },
    ],
  },
  {
    src: "https://live.staticflickr.com/65535/52567236964_10c45d02ef_b.jpg",
    original: "https://live.staticflickr.com/65535/52567236964_10c45d02ef_b.jpg",
    caption: "Louisiana Tip Jar (HTML, CSS, JS)",
    link: "https://louisianatipjar.com/",
    tags: [
      { value: "Link", title: "Link" },
    ],
  },
  {
    src: "https://live.staticflickr.com/65535/52566961951_3ef2e55517_b.jpg",
    original: "https://live.staticflickr.com/65535/52566961951_3ef2e55517_b.jpg",
    caption: "Chouest Academy (Grav CMS)",
    link: "https://chouestacademy.com/",
    tags: [
      { value: "Link", title: "Link" },
    ],
  },
  {
    src: "https://live.staticflickr.com/65535/52567405885_0be7cd2039_k.jpg",
    original: "https://live.staticflickr.com/65535/52567405885_0be7cd2039_k.jpg",
    caption: "Rig Pass Training (Bootstrap)",
    link: "https://rigpasstraining.com/",
    tags: [
      { value: "Link", title: "Link" },
    ],
  },
  {
    src: "https://live.staticflickr.com/65535/52566497877_15c94b431f_k.jpg",
    original: "https://live.staticflickr.com/65535/52566497877_15c94b431f_k.jpgg",
    caption: "CVSSA Training Curriculum (Bootstrap)",
    link: "https://cvssa.org/",
    tags: [
      { value: "Link", title: "Link" },
    ],
  },
  {
    src: "https://live.staticflickr.com/65535/52567236874_8a7d1418e2_b.jpg",
    original: "https://live.staticflickr.com/65535/52567236874_8a7d1418e2_b.jpg",
    caption: "No Good Poetry Podcast (Wordpress)",
    link: "https://www.nogoodpoetry.com/",
    tags: [
      { value: "Link", title: "Link" },
    ],
  },
  {
    src: "https://live.staticflickr.com/65535/52567406110_31c83d1520_b.jpg",
    original: "https://live.staticflickr.com/65535/52567406110_31c83d1520_b.jpg",
    caption: "Beatcases (Wordpress)",
    link: "https://beatcases.com/",
    tags: [
      { value: "Link", title: "Link" },
    ],
  },
  {
    src: "https://live.staticflickr.com/65535/52566961871_77e8374f40_b.jpg",
    original: "https://live.staticflickr.com/65535/52566961871_77e8374f40_b.jpg",
    caption: "Lucky Bean Poetry (HTML, CSS, JS)",
    link: "https://luckybeanpoetry.com/",
    tags: [
      { value: "Link", title: "Link" },
    ],
  },
];
